scapy/contrib/gtp_v2.py

```diff
@@ -1068,14 +1184,24 @@ class IE_SelectionMode(gtp.IE_Base):
 
 class IE_MMBR(gtp.IE_Base):
     name = "IE Max MBR/APN-AMBR (MMBR)"
-    fields_desc = [ByteEnumField("ietype", 72, IEType),
+    fields_desc = [ByteEnumField("ietype", 161, IEType),
```

Comment:
Why are you changing this value?

Reply:
Same comment as - https://github.com/secdev/scapy/pull/2488#discussion_r386762044